corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -458,9 +470,10 @@ def get_case_datums_basic_module(self, module, form=None):
         if not form or form.requires_case():
             datums.extend(self.get_datum_meta_module(module, use_filter=True))
 
+        case_datum = datums[-1] if datums else None
```

Comment:
I'm not sure how the datums work, but is `datums[-1]` dependent on the order items get added to the returned datum from `get_datum_meta_module`? Just thinking if this could be fragile since it's implicit that `case_datum` will always be the last item returned from `get_datum_meta_module`

Reply:
That is true. It is dependent on the fact that the last datum added is the 'case datum'. However I think that is consistent with the function of `get_datum_meta_module`. The alternative would be to change the function to return a tuple of the list and the 'case datum'.

I think for now I'm going to leave it as it is and rely on the tests to catch breaking changes.